### PR TITLE
Add maalbash caption file

### PR DESCRIPTION
**Pull request by maalbash**
Caption added

### CODE REVIEW

**Review by xwang2 — COMMENTED** [review]
Looks good.

**Review by xwang2 — APPROVED** [review]

**Comment by githubteacher** [comment]
🍔

File: _posts/0000-03-01-maalbash.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+layout: slide
+title: "Welcome to our slide deck!"
+---
+
+![bad-luck-brian](https://cloud.githubusercontent.com/assets/16547949/25400743/8ae447a2-29c1-11e7-9839-5438d8fcda7b.jpg)
+{: .center}
+
+When your PC BSOD's during class
```
Comment (githubteacher) [inline _posts/0000-03-01-maalbash.md]: Or when your screen starts flickering like in Stranger Things 😱 😭

File: _posts/0000-03-01-maalbash.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+layout: slide
+title: "Welcome to our slide deck!"
+---
+
+![bad-luck-brian](https://cloud.githubusercontent.com/assets/16547949/25400743/8ae447a2-29c1-11e7-9839-5438d8fcda7b.jpg)
```
Comment (githubteacher) [inline _posts/0000-03-01-maalbash.md:6]: Bad luck brian to get bad luck brian in a variety of memes.

**Review by githubteacher — COMMENTED** [review]
This looks great @maalbash

File: _posts/0000-03-01-maalbash.md (hunk 2 above)
Comment (maalbash) [reply _posts/0000-03-01-maalbash.md:6]: still better than Pepe the frog

File: _posts/0000-03-01-maalbash.md (hunk 1 above)
Comment (maalbash) [reply _posts/0000-03-01-maalbash.md]: No spoilers please

File: _posts/0000-03-01-maalbash.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+---
+layout: slide
+title: "Welcome to our slide deck!"
```
Comment (xwang2) [inline _posts/0000-03-01-maalbash.md:3]: Take a look here....

Comment (maalbash) [reply _posts/0000-03-01-maalbash.md:3]: looking..

**Pull request merged**